app/controllers/api/compact_index_controller.rb

```diff
@@ -30,7 +32,7 @@ def set_compact_index_cache_headers
   end
 
   def render_range(response_body)
-    headers['ETag'] = '"' << Digest::MD5.hexdigest(response_body) << '"'
+    headers['ETag'] = +'"' << Digest::MD5.hexdigest(response_body) << '"'
```

Comment:
please use `.dup`, `+` doesn't seem obvious (I don't even know about this operator :sweat_smile: ).